tutorials/sinatra/index.mdx

```diff
@@ -71,19 +72,19 @@ Done installing documentation for sinatra after 1 seconds
 
 ## Creating your first Sinatra Application
 
-1. Create a new file that you can call `testapp.rb`
+1. Create a new file that you can call `testapp.rb`.
     ```
     nano testapp.rbna
     ```
-2. Paste the following
+2. Paste the following:
     ```
     require "sinatra"
 
     get "/" do
       "OMG, a big HELLO to my first app with Sinatra!"
     end
     ```
-3.  Run your little app. The parameter `-o 0.0.0.0` will bind the application to all interfaces of the server. Without this parameter it will only be accessible from your localhost.
+3.  Run your little app. The parameter `-o 0.0.0.0` will bind the application to all interfaces of the server. Without this parameter, it will only be accessible from your localhost.
```

Comment:
```suggestion
3.  Run your Sinatra application. The `-o 0.0.0.0` parameter will bind the application to all interfaces of the server. Without this parameter, it will only be accessible from your localhost.
```